### PR TITLE
lint cleanup

### PR DESCRIPTION
**Pull request by ryanking**
Note quite to zero lint warnings, but this reduces the lints warnings by a lot.

## Test Plan
* [ ] acceptance tests

## References
*

### CODE REVIEW

File: pkg/snowflake/grant_test.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func TestWarehouseGrant(t *testing.T) {
 
 }
 
-func TestAccountGrant(t *testing.T) {
+func TestAccountGrant(t *testing.T) { // lintignore:AT003
```
Comment (cziie) [inline pkg/snowflake/grant_test.go:101]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT003: acceptance test function name should include underscore

Comment (ryanking) [reply pkg/snowflake/grant_test.go:101]: I am trying to get this ignored

File: pkg/resources/storage_integration.go [hunk 2]
```diff
@@ -153,63 +152,63 @@ func ReadStorageIntegration(data *schema.ResourceData, meta interface{}) error {
...
+	if err := d.Set("enabled", s.Enabled.Bool); err != nil {
 		return err
 	}
 
 	// Some properties come from the DESCRIBE INTEGRATION call
 	// We need to grab them in a loop
 	var k, pType string
-	var v, d interface{}
-	stmt = snowflake.StorageIntegration(data.Id()).Describe()
+	var v, unused interface{}
```
Comment (edulop91) [inline pkg/resources/storage_integration.go:174]: what is this for?

File: pkg/snowflake/future_grant.go [hunk 3]
```diff
@@ -109,7 +109,7 @@ func (gb *FutureGrantBuilder) Share(n string) GrantExecutable {
 // Grant returns the SQL that will grant future privileges on the grant to the grantee
 func (fge *FutureGrantExecutable) Grant(p string, w bool) string {
 	var template string
-	if w == true {
+	if w {
```
Comment (edulop91) [inline pkg/snowflake/future_grant.go:112]: :hugs:

**Review by edulop91 — APPROVED** [review]

**Pull request merged**